CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - sensitive_data_exposure.token_leakage_via_referer.over_https
 - sensitive_data_exposure.mixed_content.sensitive_data_disclosure
 - sensitive_data_exposure.mixed_content.requires_being_a_man_in_the_middle
+- broken_authentication_and_session_management.session_token_in_url.over_http
+- broken_authentication_and_session_management.session_token_in_url.over_https
 
```

Comment:
should we add `broken_authentication_and_session_management.session_token_in_url` in here as well?